src/Integration.Vsix/CFamily/FileConfig.cs

```diff
@@ -264,6 +264,15 @@ public Request ToRequest(string path)
                 }
             }
 
+            internal /* for testing */ static string[] GetAdditionalOptions(string options)
+            {
+                var additionalOptions = Regex.Split(options, "(?<=^[^\"]*(?:\"[^\"]*\"[^\"]*)*) (?=(?:[^\"]*\"[^\"]*\")*[^\"]*$)");
```

Comment:
Courtesy of https://stackoverflow.com/a/4780801/339720

Reply:
Can we use a compiled regex to avoid the code of creating it each time?